### ISSUE TITLE
ProVotes

### ISSUE THREAD

**Issue by LegalSmeagol** [kennytv/list-of-shame#77]
I stupidly purchased ProVotes without reading through the reviews first. Most comments are negative and highlight how the developer cannot take feedback and is quite rude towards customers; and this assessment is true.

I'll explain how I came to hate this plugin. Most vote plugins (of which are free), have a feature which allows you to view a countdown from 24h for when you may re-vote on that site. A placeholder such as %time_left_<vote_site_name_from_config>%. VotingPlugin has this feature, and is **free**.

Now let's turn face to ProVotes, a $10 plugin. Now paid plugins usually come with a feature or guarantee which differs it from free plugins; something which supports the price. This doesn't exist with ProVotes.

The 'time left' feature I stated above indeed exists in ProVotes, but for some reason, as one placeholder, one countdown which triggers whenever one of how ever many vote sites registers a vote. So let's say you have 5 vote sites, when a user votes on one, this countdown would begin; so unless a user plans to somehow press the vote button on all 5 sites simultaneously, this placeholder is utterly useless (apart from the first site the user uses of course).

I understand this is just one feature, but where so many free plugins have done right with this feature, why can a paid plugin not achieve it? And I asked, exactly why this feature, is why it is.

This is the response I received: https://i.imgur.com/rBo3IPM.png. Now from this response, the developer clearly states that "We didn't and don't currently want to keep database records for every player based on any site they vote on as it is added complexity for little benefit at this point in time." 

Firstly, what am I paying for, if not 'added complexity', does my money, or any other customers' not motivate him enough to put in actual effort? 

Secondly, "didn't and don't currently" implies he was not and is not willing to add this feature, which brings me to my next screenshot: https://i.imgur.com/0Ta61pV.png. I'm not the only person who interpreted his message here as blatant sarcasm; after all, he had, just moments ago, told me he "didn't and don't current want to keep database records". How is this acceptable behaviour from a developer speaking to a paid customer? 

Finally, after making a casual joke to break the tense situation, I was met with a threat to be removed from the channel: https://i.imgur.com/Rbj2drm.png. To which, I admit, I could have been less direct with my following message.

Generally though in these few messages, the developer came across as passive aggressive, sarcastic and as so many plugin reviews have rightly stated, unable to take criticism. 

And to conclude, here's a screenshot of my review, and the developer's very obscure recollection of the events: https://i.imgur.com/LFRiIei.png.

TL:DR - ProVotes developer is passive aggressive, sarcastic and unable to take criticism. Many reviews support this and the plugin in general lacks so many features the developer considers 'complex', yet free plugins have achieved. Worthy of being shamed I'd say.

https://www.spigotmc.org/resources/provotes-1-7-1-18-increase-player-voting.23672/

**Comment by KaspianDev** [comment]
@kennytv i'd also add [JC_Plays_MC](https://www.spigotmc.org/members/jc_plays_mc.17757/) in general, he made this, and many more "premium" plugins with below 3 star ratings

**Comment by Rexmen1** [comment]
I hate that dev from bottom of my soul, i had bought 3 of his plugins and none of them worked correctly and had issues or even setting up for first time was pain... Worst dev in spigot mc... Bumps his resource with exactly same change log for last 30 times and spigot team has done nothing against it either...

**Comment by KaspianDev** [comment]
> I hate that dev from bottom of my soul, i had bought 3 of his plugins and none of them worked correctly and had issues or even setting up for first time was pain... Worst dev in spigot mc... Bumps his resource with exactly same change log for last 30 times and spigot team has done nothing against it either...

he literally just updates to bump his api 😂

**Comment by GNosii** [comment]
> @kennytv i'd also add [JC_Plays_MC](https://www.spigotmc.org/members/jc_plays_mc.17757/) in general, he made this, and many more "premium" plugins with below 3 star ratings

Seems like he deleted his spigot page, it's giving me an "The requested member could not be found." error

**Comment by Andre601** [comment]
Account is still there

**Comment by LegalSmeagol** [comment]
> @kennytv i'd also add [JC_Plays_MC](https://www.spigotmc.org/members/jc_plays_mc.17757/) in general, he made this, and many more "premium" plugins with below 3 star ratings

+1 to this. 

I think many people share the same opinion on him and his plugins as I do. It's saddening that people like him make so much money from his half-baked plugins... in fact his plugins are hardly cooked!
